components/Panel/DelegationPanel.tsx

```diff
@@ -155,31 +146,35 @@ export function DelegationPanel() {
               </AddDelegateInputWrapper>
             )}
             {showPendingRequests &&
-              pendingRequests.map(({ delegate, transactionHash }) => (
-                <PendingRequestWrapper key={transactionHash}>
-                  <AddressWrapper>
-                    <IconWrapper>
-                      <PendingRequestIcon />
-                    </IconWrapper>
-                    <PendingRequestDetailsWrapper>
-                      <PendingRequestText>
-                        Request sent to {truncateEthAddress(delegate)}
-                      </PendingRequestText>
-                      <PendingRequestText>
-                        Waiting for approval.
-                      </PendingRequestText>
-                      <PendingRequestText>
-                        <Link
-                          href={config.makeTransactionHashLink(transactionHash)}
-                          target="_blank"
-                        >
-                          View Transaction
-                        </Link>
-                      </PendingRequestText>
-                    </PendingRequestDetailsWrapper>
-                  </AddressWrapper>
-                </PendingRequestWrapper>
-              ))}
+              pendingSentRequestsToBeDelegate.map(
```

Comment:
the only change here is formatting. kinda annoying that the diff algo sees this as a total change, but oh well